system/Commands/Database/CreateSeeder.php

```diff
@@ -0,0 +1,168 @@
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @package    CodeIgniter
+ * @author     CodeIgniter Dev Team
+ * @copyright  2019-2020 CodeIgniter Foundation
+ * @license    https://opensource.org/licenses/MIT	MIT License
+ * @link       https://codeigniter.com
+ * @since      Version 4.0.0
+ * @filesource
+ */
+
+namespace CodeIgniter\Commands\Database;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use Config\Services;
+
+/**
+ * Creates a new seeder file.
+ *
+ * @package CodeIgniter\Commands
+ */
+class CreateSeeder extends BaseCommand
+{
+	/**
+	 * The group the command is lumped under
+	 * when listing commands.
+	 *
+	 * @var string
+	 */
+	protected $group = 'Generators';
+
+	/**
+	 * The Command's name
+	 *
+	 * @var string
+	 */
+	protected $name = 'make:seeder';
+
+	/**
+	 * the Command's short description
+	 *
+	 * @var string
+	 */
+	protected $description = 'Creates a new seeder file.';
+
+	/**
+	 * the Command's usage
+	 *
+	 * @var string
+	 */
+	protected $usage = 'make:seeder [seeder_name] [options]';
+
+	/**
+	 * the Command's Arguments
+	 *
+	 * @var array
+	 */
+	protected $arguments = [
+		'seeder_name' => 'The seeder file name',
+	];
+
+	/**
+	 * the Command's Options
+	 *
+	 * @var array
+	 */
+	protected $options = [
+		'-n' => 'Set seeder namespace',
+	];
+
+	/**
+	 * Creates a new migration file with the current timestamp.
+	 *
+	 * @param array $params
+	 */
+	public function run(array $params = [])
+	{
+		helper('inflector');
+		
+		$name = array_shift($params);
+
+		if (empty($name))
+		{
+			$name = CLI::prompt(lang('Seed.nameFile'), null, 'required');
+		}
+
+		$ns       = $params['-n'] ?? CLI::getOption('n');
+		$homepath = APPPATH;
+
+		if (! empty($ns))
+		{
+			// Get all namespaces
+			$namespaces = Services::autoloader()->getNamespace();
+
+			foreach ($namespaces as $namespace => $path)
+			{
+				if ($namespace === $ns)
+				{
+					$homepath = realpath(reset($path)) . DIRECTORY_SEPARATOR;
+					break;
+				}
+			}
+		}
+		else
+		{
+			$ns = 'App';
```

Comment:
I was thinking if we can use `APP_NAMESPACE`  if it is defined and use `App` otherwise. Something like this:

`$ns = defined('APP_NAMESPACE') ? APP_NAMESPACE : 'App';`